src/Bundle/Builder/Action/ApplyTransitionAction.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Builder\Action;
+
+final class ApplyTransitionAction
+{
+    public static function create(string $name, string $route, array $routeParameters = [], array $options = []): ActionInterface
```

Comment:
I would still add transition and graph fields, as they would be usually used (looking at Sylius example)
```suggestion
    public static function create(string $name, string $route, array $routeParameters = [], string $graph = 'default', ?string $transition = null, array $options = []): ActionInterface
```

Reply:
It doesn't work with symfony workflow, cause default value is null